turbo.json

```diff
@@ -55,6 +55,7 @@
     "TELEGRAM_BOT_TOKEN",
     "TELEGRAM_BOT_START_DELAY_MS",
     "PASSPORT_CLIENT_URL",
-    "PRETIX_SYNC_DISABLED"
+    "PRETIX_SYNC_DISABLED",
+    "IS_LOCAL_HTTPS"
```

Comment:
i see you've added this, but also disabled the lint above. you have to restart your vscode for it to pick up that you've added the environment variable (annoying, I know)

Reply:
I restarted VS Code and the es-lint error went away.